pennylane/fermi/fermionic.py

```diff
@@ -86,24 +86,24 @@ def to_string(self):
 
         >>> w = FermiWord({(0, 0) : '+', (1, 1) : '-'})
         >>> w.to_string()
-        '0+ 1-'
+        'a⁺(0) a(1)'
         """
         if len(self) == 0:
             return "I"
 
         string = " ".join(
             [
-                i + j
+                "a" + j + "(" + i + ")"
                 for i, j in zip(
                     [str(i[1]) for i in self.sorted_dic.keys()], self.sorted_dic.values()
                 )
             ]
         )
-        return string
+        return string.replace("+", "\u207a").replace("-", "")
```

Comment:
Alternatively you could do something like this to avoid having to create the string and then iterate over it again to replace the `+`, `-` characters: 

```
char_map = {"+": "\u207a", "-": "" }
       string = " ".join(
             [
                 "a" + char_map[j] + "(" + i + ")"
                 for i, j in zip(
                     [str(i[1]) for i in self.sorted_dic.keys()], self.sorted_dic.values()
              ]
       )
```

Reply:
Added.